packages/back-end/src/services/users.ts

```diff
@@ -121,42 +119,40 @@ export async function trackLoginForUser({
 
   const organizationIds = organizations.map((org) => org.id);
 
-  const auditedData: UserLoginAuditableProperties = {
-    email: user.email,
-    id: user.id,
-    name: user.name || "",
-    ip,
-    userAgent,
-    os,
-    device,
-  };
-
-  const event: UserLoginNotificationEvent = {
-    object: "user",
-    event: "user.login",
-    user: {
-      type: "dashboard",
-      email: user.email,
-      id: user.id,
-      name: user.name || "",
-    },
-    data: {
-      current: auditedData,
-    },
-    projects: [],
-    tags: [],
-    environments: [],
-    // The event contains the ip, userAgent, etc. of users
-    // When marked as containing secrets, view access will be restricted to admins
-    containsSecrets: true,
-  };
-
   try {
     // Create a login event for all of a user's organizations
-    const eventCreatePromises = organizationIds.map(async (organizationId) => {
-      const emittedEvent = await createEvent(organizationId, event);
-      if (emittedEvent) new EventNotifier(emittedEvent.id).perform();
-    });
+    const eventCreatePromises = organizationIds.map((organizationId) =>
+      createEventWithPayload({
+        payload: {
+          object: "user",
```

Comment:
Is there a reason we aren't setting `objectId: user.id` here?

Reply:
Good catch thanks!